src/components/DishDetailComponent.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import { Card, CardImg, CardText, CardBody, CardTitle } from 'reactstrap';
+
+class DishDetail extends Component {
+    formatDate(date) {
+        // formats like Jan XX, XXXX
+        return new Date(date)
+                 .toLocaleDateString('en-EN', {weekday: undefined, month: 'short', day: 'numeric', year: 'numeric' })
+    }
+
+    renderComments(comments) {
+        if (comments) {
+            const renderedComments = comments.map((comment) => {
+                return (
+                    <li key={comment.id}>
+                        {/* it's technically invalid to include <div> inside <li> (because they're both block elements), but it works */}
+                        <div>{comment.comment}</div>
+                        <div className='m-2'>
+                            — {comment.author}, {this.formatDate(comment.date)}
+                        </div>
+                    </li>
+                )
+            });
+
+            return (
+                <div>
+                    <h4>Comments</h4>
+                    <ol className='list-unstyled'>
+                        {renderedComments}
+                    </ol>
+                </div>
+            )
+        } else {
+            return (
+                <div />
+            )
+        }
+    }
+
+    render() {
+        return (
```

Comment:
Hmm I think there should be some enforcement that `dish` is present on your props programmatically here other this code will error.

Also more of a nit but it's good to destructure variables at the top:

const { dish = {} } = this.props;
const { image, name, description, comments } = dish;